Otec/NewsFeedViewController.swift

```diff
@@ -41,7 +41,7 @@ class NewsFeedViewController: UIViewController, UITableViewDelegate {
     }
     
     private func retrieveNewsFromServer() {
-        NewsListener().listenToNewsWithEventSource(self.eventSource) { responseNew in
+        NewsListener().listenToNewsWithEventSource(self.eventSource, forNewspapersIDs: UserNewspaperSession.newspapersSubscribed()) { responseNew in
```

Comment:
It would be cool if we name that array `UserNewspaperSession.newspaperIDsSubscribed()`